bucoffea/scripts/run_quick.py

```diff
@@ -2,12 +2,21 @@
 
 from bucoffea.helpers.dataset import extract_year
 from bucoffea.processor.executor import run_uproot_job_nanoaod
-from bucoffea.monojet import monojetProcessor
 from bucoffea.helpers.cutflow import print_cutflow
 from coffea.util import save
 import coffea.processor as processor
+import argparse
+
+def getDatasetType():
```

Comment:
Rename to `parse_commandline` or similar to clarify that this is used to get command line arguments (might be expanded in the future)